static/src/javascripts/projects/common/modules/experiments/tests/adblock-sticky-banner.js

```diff
@@ -0,0 +1,69 @@
+define([
+    'common/utils/_',
+    'common/utils/config',
+    'common/utils/detect',
+    'common/utils/template',
+    'common/views/svgs',
+    'common/modules/adblock-banner'
+], function (
+    _,
+    config,
+    detect,
+    template,
+    svgs,
+    AdblockBanner
+) {
+
+    return function () {
+
+        this.id = 'AdblockStickyBanner';
+        this.start = '2015-08-11';
+        this.expiry = '2015-08-21';
+        this.author = 'Zofia Korcz';
+        this.description = 'Test if ad-block users are encouraged to support us by joining membership';
+        this.audience = 0.1;
+        this.audienceOffset = 0.25;
+        this.successMeasure = 'Ad-block users will be interested in becoming a Supporter';
+        this.audienceCriteria = 'Users who have ad-block installed and are not on mobile';
+        this.dataLinkNames = 'supporter message, read more';
+        this.idealOutcome = 'Users will sign up as a Supporter or will turn off the ad-block';
+
+        this.canRun = function () {
+            /**
+             * Adblock users who are not on mobile.
+             */
+            return detect.adblockInUse && detect.getBreakpoint() !== 'mobile';
+        };
+
+        this.variants = [{
+            id: 'variant',
+            test: function () {
+                var variations = [
+                    {
+                        supporterLink: 'https://membership.theguardian.com/about/supporter?INTCMP=ADBLOCK_BANNER_MONBIOT',
+                        quoteText: 'Become a Guardian Member and support independent journalism.',
```

Comment:
ths punctuation isn't consistent - the other two messages don't have a full stop